### PR TITLE
fix(api): emit dual interrupt payload fields

### LINKED ISSUE #491: [BUG] Protocol v2 input.requested is incompatible with the JavaScript SDK interrupt shape
**Describe the bug**

Aegra emits Protocol v2 `input.requested` content as `value`, while `@langchain/langgraph-sdk` expects `payload`. The event arrives live, but the JS client cannot expose its content, so `@langchain/react` does not render the input form.

Aegra's [`normalize_input_requested()`](https://github.com/aegra/aegra/blob/8cf222fe9bbf1c50d79cc484c7298c03c3d73d6b/libs/aegra-api/src/aegra_api/services/event_streaming/normalizers.py#L69) explicitly documents that `value`—not `payload`—is intentional because it matches `thread.interrupts[].value`. This matches the Python/thread representation but conflicts with the JS Protocol v2 client.

**To Reproduce**

1. Run a workflow on Aegra that calls `interrupt({...})`.
2. Subscribe to `/threads/{thread_id}/stream/events` with the JS SDK and include the `input` channel.
3. Aegra emits:

   ```json
   {
     "method": "input.requested",
     "params": {
       "data": {
         "interrupt_id": "...",
         "value": { "title": "User input requested", "prompt": "..." }
       },
       "namespace": []
     }
   }
   ```

4. The event is visible in the live stream, but the user-input form is not rendered. The interrupt remains present in thread state after reload.

**Expected behavior**

Protocol v2 HITL interrupts work with the official JavaScript LangGraph SDK.

**Actual behavior**

The JS SDK reads `params.data.payload`, so Aegra's `params.data.value` content is unavailable to it.

**Environment**

- Aegra 0.9.24
- Python `langgraph-sdk` 0.4.2
- `@langchain/langgraph-sdk` 1.9.28
- `@langchain/react` 1.0.29

Is Aegra intentionally targeting only the Python/thread interrupt representation here, or should this Protocol v2 event interoperate with both official SDKs?

### PR DESCRIPTION
**Pull request by rrajpuro**
## Description
Emit both `payload` and `value` in Protocol v2 `input.requested` events. This preserves the current Python SDK interrupt representation while restoring JavaScript SDK and React compatibility.

## Type of Change
- [x] `fix`: Bug fix
- [x] `docs`: Documentation changes
- [x] `test`: Tests added/updated

## Related Issues
Fixes #491

## Changes Made
- Add the protocol/JavaScript-facing `payload` field alongside the Python SDK-facing `value` field.
- Preserve validation and handling of missing, null, empty, and falsy interrupt values.
- Add normalizer, session projection, SSE route, and live Python SDK HITL regression coverage.
- Document the dual-field `input.requested` wire contract in the streaming guide.
- Bump `aegra-api`, `aegra-cli`, and the lockfile together from `0.10.0` to `0.10.1`.

## Testing
- [x] Unit tests added/updated
- [x] Integration tests added/updated
- [x] E2E tests added/updated
- [ ] Manual testing performed

Results:
- Targeted unit/session/integration suite: 80 passed
- LocalExecutor E2E suite: 117 passed, 3 skipped
- Python SDK HITL E2E: passed with LocalExecutor and WorkerExecutor
- Ruff and targeted type checks: passed
- `uv lock --check`: passed

## Checklist
- [x] Code follows project style (Ruff formatting)
- [x] All linting checks pass (`make lint`)
- [ ] Type checking passes (`make type-check`)
- [ ] All tests pass (`make test`)
- [x] Documentation updated
- [x] Package versions updated together
- [x] Commit messages follow the repository `[component] Brief description` format
- [x] PR title follows the repository `[component] Brief description` format

## Screenshots (if applicable)
Not applicable.

## Additional Notes
Repository-wide type checking has pre-existing diagnostics. `make ci-check` completed formatting, linting, and security checks, but its broad test phase invokes E2E tests without provisioning their required services in this local environment. No dependency constraints, resume commands, thread-state representation, or Node tooling were changed.

## Summary by CodeRabbit

* **New Features**
  * `input.requested` events now include a `payload` field alongside `value`, providing complete interrupt details.
  * Resume workflows return and validate interrupt responses for more reliable approval handling.

* **Bug Fixes**
  * Preserved interrupt payload data across streaming, session handling, and cross-run resume flows.
  * Improved handling for empty, falsy, and wrapped interrupt values.

* **Documentation**
  * Added guidance for handling human-in-the-loop interrupts over SSE, including payload formats and value behavior.

<h3>Greptile Summary</h3>

The PR restores JavaScript and React compatibility by emitting identical `payload` and `value` fields for Protocol v2 `input.requested` events while preserving Python SDK behavior.

- Documents the dual-field wire contract and missing, null, empty, and falsy-value semantics.
- Adds unit, session, integration, and live SDK regression coverage.
- Coordinates the patch release across `aegra-api`, `aegra-cli`, and `uv.lock`.

<h3>Confidence Score: 5/5</h3>

The PR appears safe to merge.

No blocking failure remains.

<details><summary><h3>Important Files Changed</h3></summary>

| Filename | Overview |
|----------|----------|
| libs/aegra-api/src/aegra_api/services/event_streaming/normalizers.py | Mirrors each present interrupt `value` into the protocol-facing `payload` field while preserving absent and falsy-value semantics. |
| docs/guides/streaming.mdx | Accurately documents the dual-field `input.requested` wire contract and edge-case behavior. |
| libs/aegra-api/tests/e2e/test_event_streaming/test_sdk_v2_e2e.py | Verifies that the live Python SDK retains its interrupt value and can resume on the same stream. |
| libs/aegra-api/tests/integration/test_api/test_event_streaming.py | Verifies the serialized SSE envelope contains identical `payload` and `value` fields. |
| libs/aegra-api/pyproject.toml | Applies the coordinated API package patch-version bump to 0.10.3. |
| libs/aegra-cli/pyproject.toml | Keeps the CLI package version synchronized with the API package at 0.10.3. |
| uv.lock | Synchronizes the two local editable package versions without changing third-party dependency versions. |

</details>

<sub>Reviews (4): Last reviewed commit: ["docs(streaming): show complete interrupt..."](https://github.com/aegra/aegra/commit/3485183bebff1667195f0ff1a7f6a27296577186) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=52622615)</sub>

**Context used:**

- Knowledge Base — [Streaming: from LangGraph execution to SSE wire events](https://app.greptile.com/aegra/-/custom-context/knowledge-base/aegra/aegra/-/docs/streaming.md)

### CODE REVIEW

File: libs/aegra-api/src/aegra_api/services/event_streaming/normalizers.py [hunk 1]
```diff
@@ -67,18 +67,15 @@ def _as_values(value: Any) -> Any:
...
-    """
+    """Project interrupts with protocol ``payload`` and Python-SDK ``value`` fields."""
     requests: list[dict[str, Any]] = []
     for entry in _interrupt_array(payload):
         if not isinstance(entry, dict) or not isinstance(entry.get("id"), str):
             continue
         request: dict[str, Any] = {"interrupt_id": entry["id"]}
         if "value" in entry:
             request["value"] = entry["value"]
+            request["payload"] = entry["value"]
```
[the remark this answers is not in the document]
Comment (rrajpuro) [reply libs/aegra-api/src/aegra_api/services/event_streaming/normalizers.py:78]: Addressed in f29cd29: documented the dual `payload`/`value` `input.requested` wire contract in the streaming guide, bumped both `aegra-api` and `aegra-cli` to `0.10.1`, and regenerated `uv.lock`. `uv lock --check`, Ruff, and the 80-test focused suite pass.

File: docs/guides/streaming.mdx [hunk 2]
```diff
@@ -311,6 +311,26 @@ data: {"type":"event","seq":3,"event_id":"...:3","method":"lifecycle","params":{
...
+  "params": {
+    "data": {
+      "interrupt_id": "interrupt-id",
+      "payload": {"question": "Approve?"},
+      "value": {"question": "Approve?"}
+    },
+    "namespace": []
+  }
+}
+```
```
[the remark this answers is not in the document]
Comment (rrajpuro) [reply docs/guides/streaming.mdx:333]: Addressed in 3065c40. The `input.requested` example now includes the complete Protocol v2 envelope metadata (`type`, `seq`, and `event_id`) alongside `method` and `params`, matching the preceding wire-format documentation.

**Comment by simon-lighthouse** [comment]
+1 this is needed to ensure interrupts function correctly in the updated langgraph package

**Review by ibbybuilds — APPROVED** [review]
@rrajpuro rebased your branch onto main to clear the version conflict (0.10.2 shipped yesterday, so this is now 0.10.3) and regenerated the lockfile, no code changes. verified locally on the rebased head: event streaming unit + integration suites 135 passed, and the full sdk v2 e2e file against a live server 10 passed including the hitl dual-field + resume test. dual emit is the right call given the protocol spec says payload and the python sdk still reads value (langchain-ai/langgraph#8572). will merge once ci is green

**Review by ibbybuilds — APPROVED** [review]
LGTM!
